main.cpp

```diff
@@ -46,103 +46,28 @@ Project 4: Part 4 / 9
  You will need to use Forward Declaration and out-of-class definitions to complete this.
  */
 
+#include <iostream>
+#include <math.h>
+
 
+struct FloatType;
+struct DoubleType;
+struct IntType;
 
 struct Point
 {
-    Point& multiply(float m)
-    {
-        x *= m;
-        y *= m;
-        return *this;
-    }
+    Point(float fx_, float fy_);
```

Comment:
These instructions aren't what you have in the project.
But they are easier to understand. 
```
3) modify the Point class below to have 3 Constructors that accept your UDTs and one that accepts primitives.
     a) make the constructor that takes primitives initialize the two member variables.
     b) for each of your 3 Ctors that accept UDTs: correctly implement a Delegating Constructor that calls the constructor which takes primitives
```
- [ ] revise